mods/cnc/maps/nod07c/rules.yaml

```diff
@@ -116,6 +116,9 @@ TRAN:
 HQ:
 	AirstrikePower:
 		Prerequisites: gdi
+		Squad@override:
+			SquadMember:
+				UnitType: a10 # Assuming we want consistency with Nod07a and Nod07b. This should be discussed during review.
```

Comment:
There is no script to send airstrikes in this mission and there is a fail trigger for killing or capturing the comm center, so this override isn't even needed. Nod07a and 07b are different missions, so no worries about consistency.